### PR TITLE
New svelte-based UI

### PR DESCRIPTION
**Pull request by will-moore**
Using https://svelte.dev/ framework to build a nicer UI.
Still using `vite` to build etc so that should still work the same 🤞 

Deployed at e.g. https://ome-ngff-validator.netlify.app/?source=https://uk1s3.embassy.ebi.ac.uk/idr/zarr/v0.1/12689244.zarr

Screenshot 2022-06-21 at 09 24 20

### CODE REVIEW

**Comment by will-moore** [comment]
@joshmoore I wonder if you could set-up netlify for `ome` to build and deploy PR branches?
I just tried it for my own repo, but it doesn't seem to build branches like this one (maybe because it's a PR to another repo).

File: package.json [hunk 1]
```diff
@@ -1,13 +1,16 @@
 {
-  "name": "ome-ngff-validator",
+  "name": "ome-ngff-validator2",
```
Comment (joshmoore) [inline package.json]: Can you explain this bump to 2? Is this temporary?

**Review by joshmoore — COMMENTED** [review]
This looks severely fancy, @will-moore! :+1:

Is Svelte something that you've been looking at elsewhere? What are your thoughts on the long-term maintainability of it?

Only change suggestion I have at the moment is that the favicon is of OMERO rather than OME.

File: package.json (hunk 1 above)
Comment (will-moore) [reply package.json]: Oops - typo. I started a new `svelte` project in https://vitejs.dev/ then copied everything over when I had it working - forgot to fix that. Fixed in [a3c5ee2](https://github.com/ome/ome-ngff-validator/pull/5/commits/a3c5ee2dc1612daa3e4efd5073b1aa39d05f629c).

**Comment by will-moore** [comment]
Thanks! I hope the background is not over-the-top (too playful)?

I've played with svelte on a side project https://idr-tables.netlify.app/ (https://github.com/will-moore/idr_tables) 
and it's really nice - it's one of the leading next-gen frameworks to follow React.

I need to play a bit more to fix the chunks on the 3D cubes and add "Tasks" and "Chunks" to the table to replicate:

Screenshot 2022-05-30 at 19 21 55

I think I understand what "Chunks" is: but what is "Tasks" and is it always `chunks + 1`?

That screenshot is from https://ome-ngff-validator.netlify.app/?source=https://uk1s3.embassy.ebi.ac.uk/idr/zarr/v0.1/9822152.zarr
Ooops - I noticed 'Bytes' is not showing up there. Will check...

**Comment by joshmoore** [comment]
> @joshmoore I wonder if you could set-up netlify for `ome` to build and deploy PR branches?

Done.

**Comment by will-moore** [comment]
Grrr - @joshmoore it looks like the deploy is failing because of some permissions. Can't access this PR/branch:

```
10:23:27 PM: git ref pull/5/head does not exist or you do not have permission
10:23:27 PM: Creating deploy upload records
10:23:27 PM: Failing build: Failed to prepare repo
10:23:27 PM: Failed during stage 'preparing repo': git ref pull/5/head does not exist
```

I wonder if there's some issue with me setting up netlify on the repo before transferring the repo to you.
I could try and Delete my netlify deploy of this?

**Comment by will-moore** [comment]
OK, I'm done with adding stuff to this branch.
Ready for review - re-deployed as above.

 - 2-column layout on desktop-sized screens (1 column on mobile)
 - Browsable JSON for top-level `.zattrs`
 - Fixed display of zarray data in table, bytes, counts etc.
 - OME Favicon

**Comment by will-moore** [comment]
Updated screenshot in description to show current state

**Comment by joshmoore** [comment]
Thanks for all the changes, @will-moore. The latest netlify build looks good, but we should try to get those building on the PRs themselves.

**Pull request merged**